Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+/// The view model for a pet sitter profile view is responsible for receiving and processing user actions
+/// to change model data and passes back the updated model data.
+final class SitterProfileViewModel: ObservableObject {
+    // MARK: - Internal interface
+
+    /// The name of the pet sitter.
+    @Published var name: String = ""
+
+    /// The surname of the pet sitter.
+    @Published var surname: String = ""
+
+    /// The phone of the pet sitter.
+    @Published var phone: String = ""
+
+    /// The additional information about the pet sitter, like his experience with dogs, favourite places for walks,
+    /// special skills, certificates, etc.
+    @Published var bio: String = ""
+
+    /// The price per hour for walking charged by the pet sitter.
+    @Published var pricePerHour: String = ""
+
+    /**
+     Initializes an instance of the `SitterProfileViewModel` class.
+     */
+    init() {
+        currentSitter = loadSitterFromStorage()
+
+        name = currentSitter.name
+        surname = currentSitter.surname
+        phone = currentSitter.phone
+        bio = currentSitter.bio
+        pricePerHour = String(currentSitter.pricePerHour)
+    }
+
+    /// Requests model layer to save modified data.
+    func save() {
+        currentSitter.name = name
+        currentSitter.surname = surname
+        currentSitter.phone = phone
+        currentSitter.bio = bio
+        currentSitter.pricePerHour = Double(pricePerHour) ?? 0
```

Comment:
I think it will be better to keep old `pricePerHour` if something go wrong.
```suggestion
                if let newPricePerHour = Double(pricePerHour) {
            currentSitter.pricePerHour = newPricePerHour
        }
```

Reply:
Thanks, fixed it.